template.rb

```diff
@@ -36,39 +39,39 @@ def apply_template!
   apply "public/template.rb"
   apply "spec/template.rb"
 
-  git :init unless preexisting_git_repo?
-  empty_directory ".git/safe"
+  # The block passed to "after_bundle" seems to run after `bundle install`
+  # but also after `webpacker:install` and after Rails has initialized the git
+  # repo
+  after_bundle do
```

Comment:
This diff is a bit messy but basically I moved lots of commands inside the `after_bundle` hook. This allowed me to remove our git repo setup and our call to `webpacker:install`.